pkg/cli/cvd.go

```diff
@@ -653,3 +738,22 @@ func uploadFiles(srv hoclient.HostOrchestratorService, uploadDir string, names [
 	}
 	return nil
 }
+
+// Deep copies src to dst using json marshaling.
+func deepCopy(src, dst any) {
+	b, err := json.Marshal(src)
+	if err != nil {
+		panic(err)
```

Comment:
Don't panic inside these utility functions, return an error instead and let the cli-specific code decide how to handle that (possibly panicking).

Reply:
Done.